types/ApiRequests.ts

```diff
@@ -194,6 +194,9 @@ export interface EditMerchItemRequest {
 export interface CreateMerchItemOptionRequest {
   option: MerchItemOption;
 }
+export interface CreateMerchItemPhotoRequest {
+  // position: number;
+}
```

Comment:
Is this request for the purpose of just having a request body since all the necessary info is passed through the uuid in the params? If so, is position:number necessary to be commented out or can we represent this better?

Reply:
It was a residue of old code - I originally created this in case I needed to include any parameters. Turns out I don't need any.

Reply:
Nice catch